### ISSUE TITLE
how to build textfusenet using version 0.4 of detectron2?

### ISSUE THREAD

**Issue by mobassir94** [ying09/TextFuseNet#59]
hi, 
i am trying to rebuild this project using rtx3090 and with detectron 0.4 version as version 0.1 is old and it doesn't work on latest ampere architecture based gpus.
i need some help from you.

question 1 : 
is there any modification possible here in this instruction : https://github.com/ying09/TextFuseNet/blob/c359cc3b1ca6f11771ea4218c11b5a8699014c1d/step-by-step%20installation.txt which will let us build this project easily on version 0.4 of detectron2? if yes then how to?

question 2 : 
this one : https://github.com/ying09/TextFuseNet/blob/master/detectron2/modeling/roi_heads/roi_heads.py#L469
is different than this one : https://github.com/facebookresearch/detectron2/blob/master/detectron2/modeling/roi_heads/roi_heads.py#L536

how do i pass cfg there?
one is pythonic and other one is nn module based.
can you help me  by giving some instructions? thank you in advance

**Comment by Real-YeJ** [comment]
Actually, the installation program is just to implement detectron2. No matter which version of detectron2, as long as you compile correctly, and then integrate the corresponding code(roi_head.py, seg_head.py, multi_path_fusion.py) to your new detectron2.

**Comment by mobassir94** [comment]
thank you for your reply @Real-YeJ 
i have installed the detectron2 correctly,i can now import detectron2 and also can import _C like this : 
from detectron2 import _C
i have also integrated seg_head.py and multi_path_fusion.py and also in configulation i set up mutil_path_fuse_on = true and all other settings like you did
but i am struggling to integrate roi_head.py

question 2 :
this one : https://github.com/ying09/TextFuseNet/blob/master/detectron2/modeling/roi_heads/roi_heads.py#L469
is different than this one : https://github.com/facebookresearch/detectron2/blob/master/detectron2/modeling/roi_heads/roi_heads.py#L536

how do i pass cfg there?

can you share the updated roi_head.py file here with me which should work with detectron2 version 0.4? it will help me a lot,i have been spending a lot of time for textfusenet,not able to run this project yet successfully

**Comment by mobassir94** [comment]
we have stopped working on this long ago. For us current best text detection model is dbresnet50. but i would like to share our last known working textfusenet version that worked for bangla text detection, here is the project : https://drive.google.com/drive/folders/1vfYmmAc9uQYDdj-aqIZtSYVANy3VSlBb?usp=sharing
i can't remember the details about how to run this project and what are the changes we made back then,check carefully to find the changes and try to understand them on your own but this project worked for us,we modified a lot of things and then this updated textfusenet worked for us with detectron2 version 0.4. best of luck.
